src/txn/txn_ckpt.c

```diff
@@ -606,6 +609,9 @@ __checkpoint_prepare(WT_SESSION_IMPL *session, const char *cfg[])
 			    sizeof(timestamp_config),
 			    "read_timestamp=%s", timestamp_buf));
 			txn_cfg[2] = timestamp_config;
+			__wt_verbose(session, WT_VERB_TIMESTAMP,
+			    "Checkpoint requested at stable timestamp: %s",
+			    timestamp_config);
```

Comment:
That makes sense to me Sulabh. I feel like the best alternative is to make this function take varargs - and that seems more complex than is worthwhile yet. If we come across a broader range of verbose timestamp messages in the future we can make the change then.